jekyll/_cci2/faq.md

```diff
@@ -7,6 +7,12 @@ categories: [getting-started]
 order: 50
 ---
 
+## Can I try out CircleCI 2.0 and still use 1.0?
+
+Yes. If you're not sure if you're ready to go all-in on 2.0 and need time to get the configuration running correctly, you can try it out while still building on 1.0. Create a new branch and remove the 1.0 `circle.yml` file from that branch. Add 2.0 configuration as described in this documentation. Now when you push the branch with 2.0 config, it will build on CircleCI 2.0. The rest of your project will continue to build on CircleCI 1.0.
+
+If you switch completely to 2.0 and then decide to go back to 1.0, all you need to do is replace the 2.0 config with 1.0 config and you'll be back to the original system.
```

Comment:
Pretty sure you don't need to remove circle.yml -- doesn't presence of  2.0 config supersede?